src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -123,6 +124,18 @@ def load_arguments(self, _):
         c.argument('sku', options_list=['--sku', '-s'], help='policy sku.', arg_type=get_enum_type(['free', 'standard']))
         c.argument('notscopes', options_list='--not-scopes', nargs='+')
 
+    with self.argument_context('policy assignment create', resource_type=ResourceType.MGMT_RESOURCE_POLICY, min_api='2018-05-01') as c:
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
+
+    with self.argument_context('policy assignment create', resource_type=ResourceType.MGMT_RESOURCE_POLICY, arg_group='Managed Identity', min_api='2018-05-01') as c:
+        c.argument('assign_identity', nargs='*', validator=validate_msi, help="Assigns a system assigned identity to the policy assignment.")
+        c.argument('identity_scope', help="Scope that the system assigned identity can access")
```

Comment:
This `--identity-scope` is different from `--scope`?  Are they used together?

Reply:
Yes, other MSI enabled resources (i.e containers, VMs, etc...) are using "--scope" to define the scope that the MSI should be given access.  Unfortunately, policy already uses "--scope" to define the scope that the policy is being assigned at.

I could assign the MSI permissions at the same scope that the policy is assigned at (which is what people will want most of the time).  However, I feel that it should err on the side of being explicit with anything that has RBAC implications.